packages/roc-package-web-app-react/app/shared/flux/create-store.js

```diff
@@ -19,9 +19,10 @@ export default function createReduxStore(reducers, ...middlewares) {
     return (callback) =>
         (history, initialState) => {
             let finalCreateStore;
+            const normalMiddlewares = [].concat(middlewares);
```

Comment:
Couldn't you use spread here? 
`const normalMiddlewares = [...middlewares, routeMiddleware(history)]`